RecoJets/JetProducers/plugins/SubEventGenJetProducer.h

```diff
@@ -11,7 +11,7 @@
  ************************************************************/
```

Comment:
This is a duplicate of  RecoJets/JetProducers/interface/SubEventGenJetProducer.h